mlir/tools/imex-opt/passes.cpp

```diff
@@ -28,13 +28,6 @@
 
 // Passes registration.
 
-static mlir::PassPipelineRegistration<>
```

Comment:
please, do no remove it from imex-opt as I'm planning to write lit test for it, LGTM othertwise